### PR TITLE
feat(scim): Change messaging for okta beta

### PR DESCRIPTION
**Pull request by JoshFerge**
We are letting all orgs who use okta enable SCIM and leaving the beta alert up for a little bit longer.

### CODE REVIEW

**Comment by JoshFerge** [comment]
not sure what's up with those other whitespace changes, they seem linter-related. will see if they have any impact on formatting

**Comment by JoshFerge** [comment]
whitespace changes seem fine. correcting one small text issue i see.

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 1]
```diff
@@ -19,22 +20,21 @@ This feature is not currently available to all Sentry users. If you are interest
 ### Requirements
 
 Okta SCIM provisioning requires:
+
 - A subscription to Sentry Business Plan or higher.
 - Configuration of SAML SSO for Okta [as documented here](/product/accounts/sso/okta-sso/), or use the Okta sign-on tab in your Sentry Okta application to configure SAML.
-- Selection of **You must select Email for the Application username format** in the Sign On application tab in Okta.
-   ![Okta username configuration](scim-username-config.png)
-
+- **You must select Email for the Application username format** in the Sign On application tab in Okta.
```
Comment (imatwawana) [inline src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:26]: Suggestion for parallel structure of the list

```suggestion
- **Selection of Email for the Application username format** in the Sign On application tab in Okta.
```

**Review by imatwawana — APPROVED** [review]
Minor smoothing, otherwise LGTM!

**Pull request merged**